src/presentation/components/header/Header.vue

```diff
@@ -26,15 +25,43 @@ const { t } = useI18n();
 const { currentRoute } = useRouter();
 
 const tabs = computed<Tab[]>(() => {
-  return [
+  const availableTabs = [
     {
       title: t('home.title'),
       path: '/',
       icon: IconPicture,
       isActive: currentRoute.value.name === 'home',
-      isPinned: true,
+      isPinned: true
     },
   ];
+
+  /**
+   * Show inactive settings tab when we are on note view
+   */
+  if ( currentRoute.value.name === 'note_view') {
+    availableTabs.push({
```

Comment:
Display settings on the Note Page instead of App Header because it is related to a particular Note